packages/@aws-cdk/aws-redshift/lib/private/database-query-provider/create-user.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable-next-line import/no-unresolved */
+import * as AWSLambda from 'aws-lambda';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import * as SecretsManager from 'aws-sdk/clients/secretsmanager';
```

Comment:
We can declare this as a devdependency and avoid this, no?

Reply:
Produces an error:
> 'aws-sdk' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies